posthog-android/src/main/java/com/posthog/android/replay/PostHogSessionReplayConfig.kt

```diff
@@ -24,4 +24,12 @@ public class PostHogSessionReplayConfig(
      */
     @PostHogExperimental
     public var captureLogcat: Boolean = true,
+
+    /**
+     * Converts custom Drawable to Bitmap
+     * By default PostHog tries to convert the Drawable to Bitmap, the supported types are
+     * BitmapDrawable, ColorDrawable, GradientDrawable, InsetDrawable, LayerDrawable, RippleDrawable
+     */
+    @PostHogExperimental
+    public var drawableConverter: PostHogDrawableConverter? = null,
```

Comment:
nice, so we do the best we can, but folk can poke in code when they're using something custom?

very cool!